ocf_data_sampler/torch_datasets/pvnet_uk_regional.py

```diff
@@ -386,12 +390,20 @@ def process_and_combine_datasets(dataset_dict: dict, config: Configuration) -> N
         )
     )
 
+    # Make sun coords NumpyBatch
+    datetimes = pd.date_range(
+        t0-minutes(gsp_config.history_minutes),
+        t0+minutes(gsp_config.forecast_minutes),
+        freq=minutes(gsp_config.time_resolution_minutes),
```

Comment:
I'm probably missing something, but I thought the idea was to divorce it from gsp overall? Correct me if I'm wrong

Reply:
Yeh after the discussion in #15, I agree its not quite right. I think we need to wait for a resolution to #31 first